GTMAppAuthSwift.podspec

```diff
@@ -31,7 +31,7 @@ requests with AppAuth.
 
   s.framework = "Security"
   s.dependency "GTMSessionFetcher/Core", "~> 1.5"
-  s.dependency "AppAuth/Core", "~> 1.4"
+  s.dependency "AppAuth/Core", "~> 1.6"
```

Comment:
Don't forget to update your Package.swift as well.

Reply:
Still not seeing Package.swift

Reply:
Sorry. Hit the request review button too early.